downloads/swan-lake-release-notes/swan-lake-2201.3.0/RELEASE_NOTE.md

```diff
@@ -0,0 +1,460 @@
+    int id;
+    string name;
+    string? department = "HR";
+};
+
+configurable Person person = ?;
+```
+
+the values can be provided in the `Config.toml` file as follows.
+
+```toml
+person = {id = 101, name = "Tom"}
+```
+
+#### Configurable support for the `json` type
+
+As `json` is a sub-type of `anydata`, it is now supported by the configurable feature.
+
+For example, if the configurable variables are defined in the following way,
+
+```ballerina
+configurable json jsonVar = ?;
+```
+
+the values can be provided in the `Config.toml` file as follows.
+
+```toml
+jsonVar = {stringValue = "string", num = 12, arr = [1, 2, "hello"], map = {a = "a"}}
+```
+
+#### Configurable support for array filler values
+
+The configurable feature is now improved to include filler values while configuring array values.
+
+For example, if the configurable variable is defined in the following way,
+
+```ballerina
+configurable int[5] arr = ?;
+```
+the values are provided in the `Config.toml` as follows.
+
+```toml
+arr = [1, 2]
+```
+It will create an array with the `[1,2,0,0,0]` elements because the filler value of the `int` type is `0`.
+
+### Bug fixes
+
+To view bug fixes, see the [GitHub milestone for 2201.3.0 (Swan Lake)](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.3.0+label%3ATeam%2FjBallerina+label%3AType%2FBug+is%3Aclosed+)
+
+## Standard library updates
+
+### New features
+
+##### `http` package
+
+- Added support for killing the application when a resource function panics 
+- Added a grace period for the graceful stop of the listener
+- Added the missing HTTP status codes
+- Made the socket configuration to be configurable in both listener and client configurations
+- Added defaultable parameter support for query parameters
+- Added support for populating the HATEOAS link's `types` field based on the resource return type
+
+#### `graphql` package
+
+- Added support for disabling introspection queries
+- Added support for GraphQL interfaces
+- Added support for interfaces implementing interfaces
+- Introduced GraphQL client configurations
+
+#### `gRPC` package
+
+- Added server reflection support for gRPC services
+
+### Improvements
+
+##### `http` package
+
+- Reduced the listener default timeout to 60s and client default timeout to 30s
+- Improved data binding based on the MIME type match
+
+#### `graphql` package
+
+- Added service-level interceptor execution for records fields, maps, and tables
+- Added service-level interceptor execution for subscriptions
+- Changed to return all the errors related to a GraphQL document in a single response
+
+#### `gRPC` package
+
+- Updated Protocol Buffers version to 3.21.7
+
+### Bug fixes
+
+To view bug fixes, see the [GitHub milestone for 2201.3.0 (Swan Lake)](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aclosed+is%3Aissue+milestone%3A%222201.3.0%22+label%3AType%2FBug)
+
+## Developer tools updates
+
+### New features
```

Comment:
@lnash94 Can you verify that we have updated all the releases notes related to the openAPI tool?

Reply:
We had to revert openAPI release note changes since the IDL feature is not going to add to update 03. Then there are no new features from openAPI tool for update 03. :)  Please find the PR https://github.com/ballerina-platform/ballerina-release/pull/2193
Already updated the link with bug fixes. https://github.com/ballerina-platform/openapi-tools/issues?q=is%3Aissue+milestone%3A%22Swan+Lake+2201.3.0%22+label%3AType%2FBug+is%3Aclosed

Reply:
Resolved as per the above comment.